docs/tables/aws_ec2_instance.md

```diff
@@ -194,4 +194,19 @@ select
   launch_template_data -> 'CapacityReservationSpecification' as capacity_reservation_specification
 from
   aws_ec2_instance;
+```
+
+### Get instance with their subnet information
+
+```sql
+select 
```

Comment:
@Pankaj-SinghR Can you please run this query and include results (with sensitive info removed) in the PR description? This helps us understand if the query is outputting the desired information.

Reply:
@cbruno10 after running following query, here is what iam getting, last columns name we haven't mentioned in query, should we name it `tag_name`  ?

```
select 
  i.instance_id, 
  i.vpc_id, 
  i.subnet_id, 
  s.tags ->> 'Name' 
from 
  aws_ec2_instance as i, 
  aws_vpc_subnet as s 
where 
  i.subnet_id = s.subnet_id;
```
| instance_id         | vpc_id                | subnet_id                | ?column?                              |
|---------------------|-----------------------|--------------------------|---------------------------------------|
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxx          | subnet-xxxxxxxx          | <null>                                |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet01 |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet01 |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxx          | subnet-xxxxxxxx          | <null>                                |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet02 |


```
select 
  i.instance_id, 
  i.vpc_id, 
  i.subnet_id, 
  s.tags ->> 'Name' as tag_name
from 
  aws_ec2_instance as i, 
  aws_vpc_subnet as s 
where 
  i.subnet_id = s.subnet_id;
```

| instance_id         | vpc_id                | subnet_id                | tag_name                              |
|---------------------|-----------------------|--------------------------|---------------------------------------|
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxx          | subnet-xxxxxxxx          | <null>                                |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet01 |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet01 |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxx          | subnet-xxxxxxxx          | <null>                                |
| i-xxxxxxxxxxxxxxxxx | vpc-xxxxxxxxxxxxxxxxx | subnet-xxxxxxxxxxxxxxxxx | test-private-public-x-PrivateSubnet02 |

Reply:
@cbruno10 Sound good, made the changes also. PTAL

Reply:
@Pankaj-SinghR Looking at the query, maybe `subnet_name` would be a slightly more descriptive alias (to avoid confusion with the instance). Can you please update it and then I think it should be good to merge!

Reply:
Hey, @cbruno10 PTAL